ui/src/components/inputs/InputsForm.vue

```diff
@@ -260,10 +260,9 @@
         {{ $t("no inputs") }}
     </el-alert>
 </template>
-<script setup>
-    import ValidationError from "../flows/ValidationError.vue";
```

Comment:
This component needs to be included in the single `script` block we're having now.